examples/CSharp/OpenGL Tutorials/Tutorial 4.1 - Model Loading/Program.cs

```diff
@@ -0,0 +1,172 @@
+        private static Vector3 CameraPosition = new Vector3(0.0f, 0.0f, 3.0f);
+        private static Vector3 CameraFront = new Vector3(0.0f, 0.0f, -1.0f);
+        private static Vector3 CameraUp = Vector3.UnitY;
+        private static Vector3 CameraDirection = Vector3.Zero;
+        private static float CameraYaw = -90f;
+        private static float CameraPitch = 0f;
+        private static float CameraZoom = 45f;
+
+        //Used to track change in mouse movement to allow for moving of the Camera
+        private static Vector2 LastMousePosition;
+
+        private static void Main(string[] args)
+        {
+            var options = WindowOptions.Default;
+            options.Size = new Vector2D<int>(800, 600);
+            options.Title = "LearnOpenGL with Silk.NET";
+            window = Window.Create(options);
+
+            window.Load += OnLoad;
+            window.Update += OnUpdate;
+            window.Render += OnRender;
+            window.Closing += OnClose;
+
+            window.Run();
+        }
+
+        private static void OnLoad()
+        {
+            IInputContext input = window.CreateInput();
+            primaryKeyboard = input.Keyboards.FirstOrDefault();
+            if (primaryKeyboard != null)
+            {
+                primaryKeyboard.KeyDown += KeyDown;
+            }
+            for (int i = 0; i < input.Mice.Count; i++)
+            {
+                input.Mice[i].Cursor.CursorMode = CursorMode.Raw;
+                input.Mice[i].MouseMove += OnMouseMove;
+                input.Mice[i].Scroll += OnMouseWheel;
+            }
+
+            Gl = GL.GetApi(window);
+
+            Shader = new Shader(Gl, "shader.vert", "shader.frag");
+            Texture = new Texture(Gl, "silk.png");
+            Model = new Model(Gl, "cube.model");
+        }
+
+        private static unsafe void OnUpdate(double deltaTime)
+        {
+            var moveSpeed = 2.5f * (float) deltaTime;
+
+            if (primaryKeyboard.IsKeyPressed(Key.W))
+            {
+                //Move forwards
+                CameraPosition += moveSpeed * CameraFront;
+            }
+            if (primaryKeyboard.IsKeyPressed(Key.S))
+            {
+                //Move backwards
+                CameraPosition -= moveSpeed * CameraFront;
+            }
+            if (primaryKeyboard.IsKeyPressed(Key.A))
+            {
+                //Move left
+                CameraPosition -= Vector3.Normalize(Vector3.Cross(CameraFront, CameraUp)) * moveSpeed;
+            }
+            if (primaryKeyboard.IsKeyPressed(Key.D))
+            {
+                //Move right
+                CameraPosition += Vector3.Normalize(Vector3.Cross(CameraFront, CameraUp)) * moveSpeed;
+            }
+        }
+
+        private static unsafe void OnRender(double deltaTime)
+        {
+            Gl.Enable(EnableCap.DepthTest);
+            Gl.Clear((uint) (ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit));
```

Comment:
```suggestion
            Gl.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);
```

People usually prefer the enum overloads